app/models/feed.rb

```diff
@@ -11,6 +11,7 @@ class Feed < ApplicationRecord
   belongs_to :saved_search, optional: true
   belongs_to :team, optional: true
 
+  before_destroy :delete_feed_teams
```

Comment:
@caiosba we already have `has_many :feed_teams` relation why not to add `dependent: :destroy` instead of adding a new callback?

Reply:
Of course! Thanks Sawy, I just made this change.